redwood-layout-composeui/src/test/snapshots/images/app.cash.redwood.layout.composeui_ComposeUiFlexContainerTest_testDynamicElementUpdates[LTR]_abcde.png

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
Why do these have lowercase suffixes? Did you start with lowercase and then switch to uppercase on a case-insensitive file system? The iOS ones have uppercase suffixes like I would expect.

Makes me wonder if we can force CI onto a case sensitive FS on Mac...

Reply:
Good catch, yeah that’s super weird. I started uppercase and I’m using a case-sensitive file system. I wonder if Paparazzi does this?!

Reply:
Confirmed.
https://github.com/cashapp/paparazzi/blob/master/paparazzi/src/main/java/app/cash/paparazzi/Snapshot.kt#L36